### PR TITLE
Updating to match dataops changes, removing refresh token

### PR DESCRIPTION
**Pull request by gregmccoy**
## Summary

Updating to match dataops changes, removing refresh token

## JIRA Issues

n/a

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Refactor or reformatting

## Testing

Are there any new or updated tests to validate the changes?

- [ ] Yes
- [x] No

## Test Directions

call PATCH api to copy files

### CODE REVIEW

File: tests/routers/v1/test_copy_approval.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def test_approve_partial_files_200(test_client, httpx_mock, mock_project):
     # mock trigger pipeline
     httpx_mock.add_response(
         method='POST',
-        url=ConfigClass.DATA_UTILITY_SERVICE + 'files/actions',
+        url=ConfigClass.DATA_UTILITY_SERVICE + 'files/actions/',
```
Comment (QXgu) [inline tests/routers/v1/test_copy_approval.py:147]: For the config cleanup I think this once should change to `DATAOPS_SERVICE`

File: tests/routers/v1/test_copy_approval.py [hunk 2]
```diff
@@ -178,7 +178,7 @@ def test_approve_all_files_200(test_client, httpx_mock, mock_project):
...
```
Comment (QXgu) [inline tests/routers/v1/test_copy_approval.py:181]: Same as above

**Review by QXgu — CHANGES_REQUESTED** [review]

**Review by QXgu — APPROVED** [review]

File: tests/routers/v1/test_copy_approval.py (hunk 1 above)
Comment (gregmccoy) [reply tests/routers/v1/test_copy_approval.py:147]: will be addressed later as part of a different ticket

File: tests/routers/v1/test_copy_approval.py (hunk 2 above)
Comment (gregmccoy) [reply tests/routers/v1/test_copy_approval.py:181]: will be addressed later as part of a different ticket

**Pull request merged**